spec/unit/printer_spec.rb

```diff
@@ -14,37 +14,6 @@
       @printer = Lhm::Printer::Percentage.new
     end
 
-    it 'prints the percentage' do
```

Comment:
`printer.notify` now just logs the message, so didn't felt much value in having tests for it.

Reply:
We should keep this test here to not break the interface for now.

Reply:
This test is still removed and not reinstated.